veracruz-server-sgx/src/veracruz_server.rs

```diff
@@ -0,0 +1,35 @@
+//! Veracruz server
```

Comment:
Same comment in this one. As far as I can tell, there is no SGX-specific code in this one.